src/work-allocation-2/containers/task-list/task-list.component.html

```diff
@@ -78,4 +78,4 @@
 
 </table>
 
-<ccd-pagination (pageChange)="paginationEvent.emit($event)"></ccd-pagination>
+<ccd-pagination *ngIf="showPagination()" (pageChange)="paginationEvent.emit($event)"></ccd-pagination>
```

Comment:
```suggestion
<ccd-pagination [autoHide]="true" (pageChange)="paginationEvent.emit($event)"></ccd-pagination>
```

Reply:
https://www.npmjs.com/package/ngx-pagination#:~:text=Default%20is%20true.-,autoHide,-%5Bboolean%5D%20If%20set

Reply:
thanks this is good solution I have taken this suggestion

Reply:
Agree with @phillip-whitaker. There seems to be an option already for this.